hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaControllerManagement.java

```diff
@@ -80,6 +80,7 @@
 import org.eclipse.hawkbit.security.SystemSecurityContext;
 import org.eclipse.hawkbit.tenancy.TenantAware;
 import org.eclipse.hawkbit.tenancy.configuration.TenantConfigurationProperties.TenantConfigurationKey;
+import org.flywaydb.core.internal.util.StringUtils;
```

Comment:
Wrong import, may I asked you to use `StringUtils` from Spring instead?

Reply:
Sure, thanks for the hint.